### PR TITLE
fixes for integration tests

### PR DESCRIPTION
**Pull request by ingwarr**
As a result of joint work on setting up a test environment with @alexkonkin, we offer several fixes.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ SHELL := /usr/bin/env bash
 # Docker build config variables
 CREDENTIALS_PATH ?= /cft/workdir/credentials.json
 DOCKER_ORG := gcr.io/cloud-foundation-cicd
-DOCKER_TAG_BASE_KITCHEN_TERRAFORM ?= 1.0.0
+DOCKER_TAG_BASE_KITCHEN_TERRAFORM ?= 20c18e5d0e8a5c669bd5306e523cc78b91cd6135
```
Comment (aaron-lane) [inline Makefile]: This should be 1.3.0.

File: test/integration/disable_client_cert/controls/gcloud.rb [hunk 2]
```diff
@@ -17,7 +17,6 @@
 cluster_name = attribute('cluster_name')
 
 credentials_path = attribute('credentials_path')
```
Comment (aaron-lane) [inline test/integration/disable_client_cert/controls/gcloud.rb]: This attribute and the associated Terraform output should be removed.

Comment (ingwarr) [reply test/integration/disable_client_cert/controls/gcloud.rb]: After disable cluster_name = attribute('cluster_name') I have a problem with kitchen verify, gktase take a look:
`kitchen verify disable-client-cert-local`
-----> Starting Kitchen (v1.24.0)
WARN: Unresolved specs during Gem::Specification.reset:
      azure_mgmt_resources (~> 0.15)
      ms_rest_azure (~> 0.11.0)
      public_suffix (< 4.0, >= 2.0.2)
      jwt (< 3.0, >= 1.4, >= 1.5)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
-----> Verifying <disable-client-cert-local>...
$$$$$$ Running command `terraform workspace select kitchen-terraform-disable-client-cert-local` in directory /home/weannye/work/gcp/forks/terraform-google-kubernetes-engine/test/fixtures/disable_client_cert
$$$$$$ Running command `terraform output -json` in directory /home/weannye/work/gcp/forks/terraform-google-kubernetes-engine/test/fixtures/disable_client_cert
disable_client_cert: Verifying

Profile: disable_client_cert
Version: (not specified)
Target:  local://

  ×  gcloud: Google Compute Engine GKE configuration
     ×  Control Source Code Error /home/weannye/work/gcp/forks/terraform-google-kubernetes-engine/test/integration/disable_client_cert/controls/gcloud.rb:21 
     undefined local variable or method `cluster_name' for #<#<Class:0x0000000005f91b68>:0x0000000005f8e1e8>


Profile Summary: 0 successful controls, 1 control failure, 0 controls skipped
Test Summary: 0 successful, 1 failure, 0 skipped
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Verify failed on instance <disable-client-cert-local>.  Please see .kitchen/logs/disable-client-cert-local.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration

File: Makefile (hunk 1 above)
Comment (ingwarr) [reply Makefile]: Done

File: test/integration/disable_client_cert/controls/gcloud.rb (hunk 2 above)
Comment (aaron-lane) [reply test/integration/disable_client_cert/controls/gcloud.rb]: @ingwarr my comment was related to `credentials_path`, not `cluster_name`.

Comment (ingwarr) [reply test/integration/disable_client_cert/controls/gcloud.rb]: Oops, sorry, it was not obvious for me

Comment (ingwarr) [reply test/integration/disable_client_cert/controls/gcloud.rb]: Done

**Comment by ingwarr** [comment]
@aaron-lane Strange behavior, it can't find jinja2 during lint tests. I'll try to reproduce it locally 
Checking file headers
The following lines have trailing whitespace
You are using pip version 10.0.1, however version 19.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Traceback (most recent call last):
  File "./helpers/generate_modules/generate_modules.py", line 21, in <module>
    from jinja2 import Environment, FileSystemLoader
ModuleNotFoundError: No module named 'jinja2'
"make generate" creates a diff, run "make generate" and commit the results
Code was generated properly

**Pull request merged**